packages/yourloops/.eslintrc

```diff
@@ -78,7 +75,8 @@
     "semi": "error",
     "lodash/prefer-lodash-typecheck": "off",
     "react-hooks/rules-of-hooks": "error",
-    "react-hooks/exhaustive-deps": "warn"
+    "react-hooks/exhaustive-deps": "warn",
+    "react/prop-types": "off"
```

Comment:
```suggestion
```
Silent a rule just because there is errors displayed is not the only solution ;-)
It's better to fix the problem.

Reply:
Is this rule relevant ?

Reply:
I think yes, I don't see why there is a problem on your functions, and not on others.

Reply:
I got the error on password.tsx and i don't get why, some help would be appreciated 😄 
`/dev/blip/packages/yourloops/components/utils/password.tsx
  51:62   error  'id' is missing in props validation          react/prop-types
  51:66   error  'label' is missing in props validation       react/prop-types
  51:73   error  'value' is missing in props validation       react/prop-types
  51:80   error  'error' is missing in props validation       react/prop-types
  51:87   error  'helperText' is missing in props validation  react/prop-types
  51:99   error  'style' is missing in props validation       react/prop-types
  51:106  error  'setState' is missing in props validation    react/prop-types`

Reply:
OK this is a known bug on prop-types. Fixed